spec/specification/dsl_spec.rb

```diff
@@ -368,6 +368,27 @@ module Pod
 
     #-----------------------------------------------------------------------------#
 
+    describe 'Test specs' do
+      before do
+        @spec = Spec.new do |spec|
+          spec.name = 'Spec'
+          spec.test_spec do |test_spec|
+            test_spec.test_type = :unit
```

Comment:
Removed it now as the attribute now has a `default_value` set when used with a `consumer.rb.`